index.html

```diff
@@ -730,7 +506,7 @@ <h3>Create and populate a ComputePressureEntry</h3>
         Let |entry:ComputePressureEntry| be a new {{ComputePressureEntry}} instance.
       </li>
       <li>
-        Let |entry|.|cpuSpeed:double| be the result of invoking [[[#handle-cpu-frequency]]] with
+        Let |entry|.|state:PressureState| be the result of invoking ... with
```

Comment:
Is `...` the spread syntax? If so, you may want to explain what this is doing exactly to avoid confusion.

Reply:
This was mostly because we will have to write that whole section in the future... I could add a TODO

Reply:
added

Reply:
`(future algo)` is not clear. Can you file an issue and link from there?

Reply:
Sure

Reply:
https://github.com/WICG/compute-pressure/issues/46